spec/integration/post_integration_spec.rb

```diff
@@ -0,0 +1,109 @@
+require 'rails_helper'
+
+RSpec.describe 'Index/Show Posts page.', type: :system do
+  subject!(:user) { User.where(name: 'Tony').first }
+  subject!(:post) { Post.where(id: 2).first }
+
```

Comment:
- [ ] 4 tests are failing, please make sure all tests pass successfully.
<img width="914" alt="Screenshot 2022-08-24 at 20 03 48" src="https://user-images.githubusercontent.com/77038610/186479410-66baaf1a-4c71-47f5-bdc1-8a0dd5a2511d.png">

Reply:
Hello @ShoiraTa !
Since im runing windows, those test pass when the rails server is running, otherwise gives me this error 
![image](https://user-images.githubusercontent.com/99830261/186480423-438984ba-5e01-4aa2-af47-274eaa7b2063.png)

Can you confirm?

Reply:
hi @alexpuente0 ,
I started server and run the specs, and still getting same errors, I checked a couple of them:

1.
```
    it 'Shows user\'s profile picture' do
      image = page.all('img')
      expect(image[0]['src']).to eq('https://cdn.icon-icons.com/icons2/1378/PNG/512/avatardefault_92824.png')
    end
```

In test    `undefined method `[]' for nil:NilClass`, means that image is undefined, please check the image  on this line `image = page.all('img')`, it is better to target an image of particular user that you create `before action` and the url expected should better be dynamic, hardcoded values can lead to error

2.  
TEST: 
```
   it 'Shows all user\'s usernames' do
      expect(page).to have_content('Peter')
      expect(page).to have_content('Tony')
      expect(page).to have_content('Clint')
    end
```


returns error expected to find text `"Peter" in "You need to sign in or sign up before continuing.\nLog in\nEmail\nPassword\nRemember me\nForgot your password?"` here you have to log in first before `expecting` .

So please check the message of error and try to address it.